Jenkinsfile.release

```diff
@@ -0,0 +1,79 @@
+#!groovy
+
+node {
+    properties([
+        parameters([
+            string(
+                defaultValue: '',
+                description: 'SHA of an existing container image. Usually what\'s currently on staging.', 
+                name: 'GIT_COMMIT',
+                trim: false
+            )
+        ])
+    ])
+
+    try {
+        // Checkout the proper revision into the workspace.
+        stage('checkout') {
+            checkout([
+                $class: 'GitSCM',
+                branches: [[name: params.GIT_COMMIT]],
+                extensions: [[$class: 'PruneStaleBranch']],
+                userRemoteConfigs: [[
+                    credentialsId: 'azaveaci',
+                    url: 'git@github.com:open-apparel-registry/open-apparel-registry.git'
+                ]]
+            ])
+        }
+
+        env.AWS_PROFILE = 'open-apparel-registry'
+        env.AWS_DEFAULT_REGION = 'eu-west-1'
+
+        env.OAR_SETTINGS_BUCKET = 'openapparelregistry-testing-config-eu-west-1'
+
+        // If we don't do this, the showmigrations stage will throw
+        // django.core.exceptions.ImproperlyConfigured
+        stage('setup') {
+            wrap([$class: 'AnsiColorBuildWrapper']) {
+                sh './scripts/bootstrap'
+            }
+        }
+
+        env.OAR_SETTINGS_BUCKET = 'openapparelregistry-production-config-eu-west-1'
+
+        // Plan and apply the current state of the production infrastructure
+        // as outlined by whatever SHA is passed through as a build parameter.
+        stage('infra') {
+            // Environment used in Rollbar deploy notifications.
+            // https://docs.rollbar.com/reference#post-deploy
+            env.OAR_DEPLOYMENT_ENVIRONMENT = 'production'
+
+            wrap([$class: 'AnsiColorBuildWrapper']) {
+                sh 'docker-compose -f docker-compose.ci.yml run --rm terraform ./scripts/infra plan'
+                withCredentials([[$class: 'StringBinding',
+                                credentialsId: 'OAR_ROLLBAR_ACCESS_TOKEN',
+                                variable: 'OAR_ROLLBAR_ACCESS_TOKEN']]) {
+                    sh 'docker-compose -f docker-compose.ci.yml run --rm terraform ./scripts/infra apply'
+                }
+            }
+        }
+
+        // Kick off an ECS task to display any outstanding migrations.
+        stage('showmigrations') {
+            wrap([$class: 'AnsiColorBuildWrapper']) {
+                sh './scripts/manage ecsmanage -e production showmigrations'
+            }
+        }
+    } catch (err) {
+        // Some exception was raised in the `try` block above. Assemble
+        // an appropirate error message for Slack.
+        def slackMessage = ":jenkins-angry: *Open Apparel Registry (Release Pipeline) (${params.GIT_COMMIT}) #${env.BUILD_NUMBER}*"
+
+        slackMessage += "\n<${env.BUILD_URL}|View Build>"
+        //slackSend channel: '#oar', color: 'danger', message: slackMessage
```

Comment:
Reminder to uncomment this before merging.

Reply:
It might be valuable to include another message earlier in the `Jenkinsfile` that signals a production release is underway.

Reply:
Because Rollbar lets you report the status of a deploy (`started`, `succeeded`, `failed`, or `timed_out`), I wanted to see if we could lean on them entirely for Slack notifications.

I added a function called `reportDeploy(String status);`, and got rid of the existing Rollbar stuff in `infra`.

Then, you could do stuff like:

```groovy
try { 
    stage('infra') {
        ...
    }
} catch(err) {
    reportDeploy("failure");
}
```

The problem is that Rollbar only seems to report when a deploy succeeds. Reporting another status seems to only populate metadata, which can be seen here: https://rollbar.com/OpenApparelRegistry/OpenApparelRegistry/deploys/.

See: https://docs.rollbar.com/reference#post-deploy